lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -675,6 +676,17 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  createTreeWalker(root, whatToShow, filter) {
+    if (!("_ownerDocument" in root)) throw new TypeError("First argument to createTreeWalker must be a Node");
+    if (whatToShow === undefined) whatToShow = 0xFFFFFFFF;
```

Comment:
Default parameters actually weren't implemented in webidl2js until just now. I thought something with the conversion was off :smile: 

You should be able to remove that setting now with webidl2js updated to 4.3.0.